src/components/HighlightedText.tsx

```diff
@@ -21,13 +18,7 @@ const HighlightedText: FC<{ value: string; match: string; disabled?: boolean }>
 
         return (
           <span key={i}>
-            <span
-              style={{
-                color: !disabled && isMatch ? colors.vividHighlight : undefined,
-              }}
-            >
-              {char}
-            </span>
+            <span className={css({ color: !disabled && isMatch ? 'vividHighlight' : undefined })}>{char}</span>
```

Comment:
Just a note for posterity: I thought that dynamic styles like these needed to be passed to `style` instead of generating a className, but it looks like Panda is able to generate multiple classNames for conditionals like this.

https://panda-css.com/docs/guides/dynamic-styling#runtime-conditions

I tested it out on another component, and it does indeed generate multiple classNames and switch between them as expected. Very cool.

Reply:
Yes, I went through the same discovery process! As long as the resulting value is static (in this case the string 'vividHighlight' or undefined), then PandaCSS handles the ternary just fine.